src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Properties/InterceptedProjectProperties/DebugPropertyPage/RemoveStartArgumentsActionHandler.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+using System.Xml;
+using Microsoft.VisualStudio.ProjectSystem.VS.PropertyPages.Designer;
+
+namespace Microsoft.VisualStudio.ProjectSystem.Properties;
+
+[Export(typeof(ILinkActionHandler))]
+[ExportMetadata("CommandName", "RemoveStartArguments")]
+internal class RemoveStartArgumentsActionHandler : ILinkActionHandler
+{
+    private const string PropertyNameToDelete = "StartArguments";
+    private const string UserSuffix = ".user";
+    private string? _xmlFile;
+
+    public Task HandleAsync(UnconfiguredProject project, IReadOnlyDictionary<string, string> editorMetadata)
+    {
+        _xmlFile = project.FullPath + UserSuffix;
+
+        _ = project.Services.ThreadingPolicy.JoinableTaskFactory.RunAsync(RemoveElementAsync);
+
+        return Task.CompletedTask;
+    }
+
+    private Task RemoveElementAsync()
+    {
+        XmlDocument doc = new XmlDocument() { XmlResolver = null };
```

Comment:
You can avoid the repeated type name here if you like:

```suggestion
        XmlDocument doc = new() { XmlResolver = null };
```

Reply:
fixed